lib/ble.toit

```diff
@@ -115,13 +119,20 @@ class AdvertisementData:
   */
   connectable/bool
 
-  constructor --.name=null --.service_classes=[] --.manufacturer_data=#[] --.connectable=false:
+  /**
+  Advertise flags. This must be a bitwise or of the BLE_ADVERTISE_FLAG_* constants
+    (see $BLE_ADVERTISE_FLAGS_GENERAL_DISCOVERY and similar).
+  */
+  flags/int
+
+  constructor --.name=null --.service_classes=[] --.manufacturer_data=#[]
+              --.connectable=false --.flags=0 --check_size=true:
```

Comment:
Nit: This is generally applicable. I'm not entirely sure about this, but I don't think we infer parameter types from their default value just yet. 

```suggestion
              --.connectable=false --.flags=0 --check_size/bool=true:
```